src/v2/guide/components-registration.md

```diff
@@ -122,20 +122,20 @@ export default {
 }
 ```
 
-Note that in ES2015+, placing a variable name like `ComponentA` inside an object is shorthand for `ComponentA: ComponentA`, meaning the name of the variable is both:
+Note que en ES2015+, colocar un nombre de variable como `ComponentA` dentro de un objeto es la abreviatura de `ComponentA: ComponentA`, lo que significa que el nombre de la variable es ambos:
```

Comment:
In my opinion, `ambos` looks a bit weird to the reader, what do you think about using `es tanto` instead?
```suggestion
Note que en ES2015+, colocar un nombre de variable como `ComponentA` dentro de un objeto es la abreviatura de `ComponentA: ComponentA`, lo que significa que el nombre de la variable es tanto:
```

Reply:
I understand that. But reading this with `ambos` looks a bit odd, at least to me 🤷‍♂ 
@sxcamacho @FLarra can you guys weigh in on this?

Reply:
In opinion, it could sound weird, but it's ok. When we talk in Spanish, sometimes we say `both` before say which both.  Makes sense?